cpp/test/comm/CMakeLists.txt

```diff
@@ -28,27 +28,45 @@ function (add_unit_test)
     cxx_target_compile_options (AppleClang ${name} PRIVATE -DBOOST_ASIO_HAS_STD_CHRONO)
 endfunction()
 
-
 # Build common code into its own library.
```

Comment:
nit: misleading comment if we follow the other two Cmakelists.txt as the standard. "shared generated code"